app/controllers/projects_controller.rb

```diff
@@ -258,18 +225,20 @@ def experiment_archive
     current_team_switch(@project.team)
   end
 
+  def dt_state_load
+    render json: { state: current_team.current_view_state(current_user).state.dig('projects', 'table') }
```

Comment:
What if there is no current_team?

Reply:
You are not able to access that screen if you do not have current_team...

Reply:
Orr.. hm, should I return 403 or something?

Reply:
Maybe just safe navigation?